nixos/modules/services/web-apps/librespeed.nix

```
@@ -0,0 +1,438 @@
+{
+  config,
+  lib,
+  options,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.librespeed;
+  opt = options.services.librespeed;
+
+  settingsFormat = pkgs.formats.toml { };
+
+  configFile = settingsFormat.generate "librespeed-rust-config.toml" cfg.settings;
+
+  librespeedAssets =
+    pkgs.runCommand "librespeed-assets"
+      {
+        serversList = ''
+          function get_servers() {
+            return ${builtins.toJSON cfg.frontend.servers}
+          }
+          function override_settings () {
+          ${lib.pipe cfg.frontend.settings [
+            (lib.mapAttrs (name: val: "  s.setParameter(\"${builtins.toJSON name},${builtins.toJSON val});"))
```

I think there's a double quote leftover here:

```suggestion
            (lib.mapAttrs (name: val: "  s.setParameter(${builtins.toJSON name},${builtins.toJSON val});"))
```

---

Just noticed that, too, while testing 😂